docs/v3.x/getting-started/react.md

```diff
@@ -0,0 +1,186 @@
+# Getting Started with React
+
+This integration guide is the following of the [Getting started](../getting-started/quick-start.html). We assume that you have completed the [Step 8](../getting-started/quick-start.html#_8-consume-the-content-type-s-api) and therefore can consume the API by browsing this [url](http://localhost:1337/restaurants).
```

Comment:
This integration guide is following the [Getting started guide].....

We assume that you have completed [Step 8]....